owners/src/parser.js

```diff
@@ -66,9 +81,19 @@ class OwnersParser {
    */
   async parseAllOwnersRules() {
     const ownersPaths = await this.localRepo.findOwnersFiles();
-    return ownersPaths
-      .map(ownersPath => this.parseOwnersFile(ownersPath))
-      .filter(rule => rule !== null);
+    const allRules = [];
+    const allErrors = [];
+
+    ownersPaths.forEach(ownersPath => {
+      const {rules, errors} = this.parseOwnersFile(ownersPath);
+      allRules.push(...rules);
+      allErrors.push(...errors);
+    });
+
+    return {
+      rules: allRules,
```

Comment:
this makes it not nullable then right? in the definition it states it can be nullable

Reply:
Removed `null|` from typedef; resolving.